source/containers.tex

```diff
@@ -1632,44 +1661,68 @@
 
 \pnum
 In \tref{container.assoc.req},
+\begin{itemize}
+\item
 \tcode{X} denotes an associative container class,
+\item
 \tcode{a} denotes a value of type \tcode{X},
+\item
 \tcode{a2} denotes a value of a type with nodes compatible with type
 \tcode{X} (\tref{container.node.compat}),
+\item
 \tcode{b} denotes a possibly \tcode{const} value of type \tcode{X},
+\item
 \tcode{u} denotes the name of a variable being declared,
+\item
 \tcode{a_uniq} denotes a value of type \tcode{X}
 when \tcode{X} supports unique keys,
 \tcode{a_eq} denotes a value of type \tcode{X}
```

Comment:
Are we intentionally keeping `a_uniq` and `a_eq` together in a single bullet, or was this overlooked? I find it inconsistent that `a_tran` is separate but these two share a bullet.

Reply:
This was overlooked. Fixed + force-pushed.